rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/BaseTypeController.java

```diff
@@ -38,6 +38,9 @@ public BaseTypeController(FieldTypeRepository fieldTypeRepository,
         @ApiResponse(code = 200, message = "All valid base types")
     })
     public List<FieldType> getBaseTypes() {
+        if (baseTypes == null) {
```

Comment:
this is not thread safe. Not a big deal but it's possible that multiple concurrent threads end up fetching the data from the db

Reply:
We will update later with few other best options to deal with concurrent threads. Thank you.

Reply:
DONE, Please, can you review. Thank you.